src/Analyzers/MSTest.Analyzers/Resources.resx

```diff
@@ -436,4 +436,13 @@
   <data name="UseParallelizeAttributeAnalyzerTitle" xml:space="preserve">
     <value>Explicitly enable or disable tests parallelization</value>
   </data>
+  <data name="TypeContaningTestMethodShouldBeATestClassTitle" xml:space="preserve">
+    <value>Type containing '[TestMethod]' must be a '[TestClass]'</value>
+  </data>
+  <data name="TypeContaningTestMethodShouldBeATestClassDescription" xml:space="preserve">
+    <value>Type contaning '[TestMethod]' must be a '[TestClass]'.</value>
```

Comment:
Typo:

```suggestion
    <value>Type containing '[TestMethod]' must be a '[TestClass]'.</value>
```

Reply:
I would add some details here to help users understand why.

Reply:
maybe: ", otherwise the test method will be silently ignored." ?